GitTrends/Views/Repository/InformationButton.cs

```diff
@@ -112,22 +115,29 @@ class FloatingActionTextButton : FloatingActionButtonView
             readonly FloatingActionButtonType _floatingActionButtonType;
 
             public FloatingActionTextButton(in MobileSortingService mobileSortingService,
-                                            in FloatingActionButtonSize floatingActionButtonSize,
                                             in FloatingActionButtonType floatingActionButtonType,
-                                            in ICommand? command = null)
+                                            in ICommand? command = null,
+                                            in FloatingActionButtonSize floatingActionButtonSize = FloatingActionButtonSize.None
+                                            )
             {
                 _mobileSortingService = mobileSortingService;
                 _floatingActionButtonType = floatingActionButtonType;
 
                 ThemeService.PreferenceChanged += HandlePreferenceChanged;
 
-                Size = floatingActionButtonSize;
+                // Override the size if the it's been specified in argument
+                if (floatingActionButtonSize != FloatingActionButtonSize.None)
+                    Size = floatingActionButtonSize;
+
+
+                //Size = floatingActionButtonSize;
                 Command = command;
 
-                var fontSize = floatingActionButtonSize switch
+                var fontSize = Size switch
                 {
                     FloatingActionButtonSize.Mini => 10,
                     FloatingActionButtonSize.Normal => 13,
```

Comment:
Since `Normal` and `Large` use the same font size, let's use some cool C# 9.0 syntax here!

```
FloatingActionButtonSize.Normal or FloatingActionButtonSize.Large => 13,
```